README.md

```diff
@@ -903,6 +903,7 @@ See also <https://en.wikipedia.org/wiki/List_of_streaming_media_systems>, <https
 - [Icecast 2](http://www.icecast.org/) - streaming audio/video server which can be used to create an Internet radio station or a privately running jukebox and many things in between. ([Source Code](https://git.xiph.org/?p=icecast-server.git;a=summary), [Clients](http://www.icecast.org/apps/)) `GPL-2.0` `C`
 - [Jellyfin](https://jellyfin.media) - Streaming audio/video server with a slick UI and robust transcoding capabilities (fork of Emby). ([Source Code](https://github.com/jellyfin/jellyfin)) `GPL-2.0` `C#`
 - [MistServer](http://mistserver.org/) - Streaming media server that works well in any streaming environment. ([Source Code](https://github.com/DDVTECH/mistserver)) `AGPL-3.0` `C++`
+- [PimpMyStremio](https://github.com/sungshon/PimpMyStremio) - PimpMyStremio is a self-hosted add-on manager for Stremio. It was created by the community to have an environment of building, updating and using local add-ons. The end result being tens of more add-ons and countless more streams that would not normally be available. `MIT` `Nodejs`
 - [ReadyMedia](http://sourceforge.net/projects/minidlna/) - Simple media server software, with the aim of being fully compliant with DLNA/UPnP-AV clients. Formerly known as MiniDLNA. ([Source Code](https://sourceforge.net/p/minidlna/git/ci/master/tree/)) `GPL-2.0` `C`
 - [Rygel](https://wiki.gnome.org/action/show/Projects/Rygel) - Rygel is a UPnP AV MediaServer that allows you to easily share audio, video, and pictures. Media player software may use Rygel to become a MediaRenderer that may be controlled remotely by a UPnP or DLNA Controller. ([Source Code](https://gitlab.gnome.org/GNOME/rygel/)) `GPL-3.0` `C`
 - [üWave](http://u-wave.net/) `⚠` - üWave is a self-hosted collaborative listening platform. Users take turns playing media—songs, talks, gameplay videos, or anything else—from a variety of media sources like YouTube and SoundCloud. ([Demo](https://wlk.yt/), [Source Code](https://github.com/u-wave)) `MIT` `Nodejs`
```

Comment:
Hi, thank you but I don't think this qualifies for the list: it's an addon manager for a closed-source, local [desktop client](https://www.stremio.com/) for a proprietary streaming platform. The list is about Free/libre, self-hosted network services/web applications - which neither Stremio or PimpMyStremio are (the fact that it runs in a web browser doesn't make it a "self-hosted" service). Unless I missed something, or you can give a similar example that is already in the list?

Clsoing, feel free to reopen if you think otherwise.

Reply:
Actually, i do believe this is by definition self-hosting, but i also understand how it could not be understood as such on first glance.

I can only explain how it all works and you can judge if it fits the list or not.

The fact that Stremio is a closed-source, local desktop client is true to an extent. I think most of it (if not all of it) is open source by now (sources: [github](https://github.com/stremio/), [blog post](https://blog.stremio.com/announcing-stremio-core-embracing-open-source-and-rust/)), and it is not only a desktop client, it also has an Android client and a Web client. The [web client](http://app.strem.io/shell-v4.4/#/) is public, and as all it's other clients free. The difference between the web client and other clients is lack of torrent and limited video encoding support.

Stremio only supports remote add-ons, which means each add-on uses a sdk that creates a web service and needs to be deployed online. There is no concept of a local add-on in Stremio. Now this is where PimpMyStremio comes in, as it's an application that literally self-hosts the add-ons ran in it, so it's not just a web GUI, it's a complete web service that handles many diverse requests based on Stremio's [add-on protocol](https://github.com/Stremio/stremio-addon-sdk/blob/master/docs/protocol.md).

PimpMyStremio currently hosts 40 unique add-ons, that can't be found anywhere else. And 37 of them can be used without having Stremio installed at all, by just pressing a button to load them in Stremio Web. The remaining 3 add-ons would need a local version of Stremio installed as they provide streams with diverse video encodings or torrents.

This means that PimpMyStremio isn't reliant on any other application to work, it can work in almost it's entirety with just Stremio Web (which is a public website)

PimpMyStremio also includes an internal proxy that it makes available to it's add-ons, it has seamless compatibility with a normal remote add-on, and it also gives the option of making it's server available online (and setting a password to it).. So it can also be used with web clients (or installed clients) remotely, for example: using it on your phone, while the PimpMyStremio is installed on your home PC.

So correct me if I'm wrong, but is this not an application for a self-hosted service? That does not actually require any other local application in order to work?

Reply:
Maybe a better description for it is:

An open source, community built self-hosted Stremio add-ons server. :))